src/utilities/util_verify.c

```diff
@@ -128,12 +145,16 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
           "-u (unredact all data), should not be set to true simultaneously with -k (unredact only "
           "keys)");
 
+    if (dump_offsets != NULL && ckpt != NULL)
+        WT_RET_MSG((WT_SESSION_IMPL *)session, ENOTSUP, "%s",
+          "-u dump_offsets, should not be set simultaneously with -v checkpoint-name");
```

Comment:
should this say `-d` for `dump_offsets` and `-C` for `checkpoint`?

Reply:
Good catch, thanks!

```suggestion
          "-d dump_offsets, should not be set simultaneously with -C checkpoint-name");
```